src/dosbox.cpp

```diff
@@ -917,9 +917,18 @@ void DOSBOX_Init() {
 	Pmulti_remain->GetSection()->Add_string("parameters", when_idle, "");
 	Pmulti_remain->Set_help(
 	        "set type of device connected to com port.\n"
-	        "Can be disabled, dummy, modem, nullmodem, directserial.\n"
+	        "Can be disabled, dummy, serialmouse, modem, nullmodem, directserial.\n"
 	        "Additional parameters must be in the same line in the form of\n"
 	        "parameter:value. Parameter for all types is irq (optional).\n"
+	        "for serialmouse: type, can be one of:\n"
+	        "                 2btn  : 2 buttons, Microsoft serial mouse\n"
+	        "                 3btn  : 3 buttons, Logitech serial mouse\n"
+	        "                 wheel : 3 buttons + wheel serial mouse\n"
+	        "                 msm   : 3 buttons, Mouse Systems Mouse\n"
+	        "                 2btn+msm, 3btn+msm, wheel+msm : autoselection\n"
+	        "                 rate, can be normal or smooth (more frequent\n"
+	        "                 mouse updates than original hardware allows)\n"
+	        "                 Default is type:wheel+msm rate:smooth\n"
```

Comment:
It would be better to reformat this whole help text to the format used by more recently introduced options. That would give you much more horizontal space for the explanation text. E.g.:

```
#       capture_mouse: Choose a mouse control method:
#                         onclick:        Capture the mouse when clicking any button in the window.
#                         onstart:        Capture the mouse immediately on start.
#                         seamless:       Let the mouse move seamlessly; captures only with middle-click or hotkey.
#                         nomouse:        Hide the mouse and don't send input to the game.
#                      Choose how middle-clicks are handled (second parameter):
#                         middlegame:     Middle-clicks are sent to the game.
#                         middlerelease:  Middle-click will release the captured mouse, and also capture when seamless.
#                      Defaults (if not present or incorrect): seamless middlerelease
#                      Possible values: seamless, onclick, onstart, nomouse.
```